RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -2,33 +2,41 @@
 @using System.Globalization
 @inject IBillRepository billRepository
 
-@section Styles {
-<link rel="stylesheet" href="/css/BillDetails.css" />
-}
 @{
     var billId = Guid.Parse(RouteData.Values["guid"].ToString());
     if (billRepository.GetById(billId) is null)
     {
-        <div id="errorText" style="color:red"> Bill was not found!</div>
+        <div class="alert alert-danger text-center">
+            Bill was not found!
+        </div>
     }
     else
     {
+        <h1 class="text-center">Bill Details</h1>
         var item = billRepository.GetById(billId);
-        <div class="bill-summary">
-            <div t-id="description">@item.Description</div>
-            <div t-id="personWhoPaid">Paid by: @(item.EntryLines[0].Person.FirstName)
-                @(item.EntryLines[0].Person.LastName) </div>
-            <div t-id="date">@item.Date.ToString("D")</div>
-            <div t-id="amount">Total amount: @item.EntryLines[0].Amount€</div>
+        <div class="p-3 border rounded mx-auto">
+            <dl class="row">
+                <dt class="col-12 col-sm-6">Description:</dt>
+                <dd t-id="description" class="col-12 col-sm-6 text-sm-end">@item.Description</dd>
+
+                <dt class="col-12 col-sm-6">Paid by:</dt>
+                <dd t-id="personWhoPaid" class="col-12 col-sm-6 text-sm-end">@item.EntryLines[0].Person.FirstName @item.EntryLines[0].Person.LastName</dd>
+
+                <dt class="col-12 col-sm-6">Date:</dt>
+                <dd t-id="date" class="col-12 col-sm-6 text-sm-end">@item.Date.ToString("D")</dd>
+
+                <dt class="col-12 col-sm-6">Total amount:</dt>
+                <dd t-id="amount" class="col-12 col-sm-6 text-sm-end">@item.EntryLines[0].Amount€</dd>
+            </dl>
         </div>
```

Comment:
different background color

Reply:
As I said before Patrick said that we should try to use no css styling but bootstrap doesn't offer many color variations. I can change the background color but I would have to use css styling in that case.